### ISSUE TITLE
SAN can not be a wildcard domain

### ISSUE THREAD

**Issue by arkadijs** [containous/traefik#4585]
For the following config:
```
[[acme.domains]]
  main = "arkadi.dev.superhub.io"
  sans = ["app.arkadi.dev.superhub.io", "apps.arkadi.dev.superhub.io", "*.app.arkadi.dev.superhub.io", "*.apps.arkadi.dev.superhub.io"]
```
An error is emitted:
> Unable to obtain ACME certificate for domains \"arkadi.dev.superhub.io,app.arkadi.dev.superhub.io,apps.arkadi.dev.superhub.io,*.app.arkadi.dev.superhub.io,*.apps.arkadi.dev.superhub.io\" : unable to generate a certificate in ACME provider for domains \"arkadi.dev.superhub.io,app.arkadi.dev.superhub.io,apps.arkadi.dev.superhub.io,*.app.arkadi.dev.superhub.io,*.apps.arkadi.dev.superhub.io\": SAN \"*.app.arkadi.dev.superhub.io\" can not be a wildcard domain

But, LE/ACME supports wildcards in SAN.
Please see attached certificate issued by Staging.

```
X509v3 Subject Alternative Name: 
  DNS:*.app.arkadi.dev.superhub.io, DNS:*.apps.arkadi.dev.superhub.io, DNS:app.arkadi.dev.superhub.io, DNS:apps.arkadi.dev.superhub.io
```
```
-----BEGIN CERTIFICATE-----
MIIFqDCCBJCgAwIBAgITAPqkg8w1yOD7lPhDdSXk4XAJVjANBgkqhkiG9w0BAQsF
ADAiMSAwHgYDVQQDDBdGYWtlIExFIEludGVybWVkaWF0ZSBYMTAeFw0xOTAzMTMx
MjM0MjFaFw0xOTA2MTExMjM0MjFaMCUxIzAhBgNVBAMTGmFwcC5hcmthZGkuZGV2
LnN1cGVyaHViLmlvMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEArhZu
XBtnQfvogMb1MS8rL2zm65LemiZfir4W6mJclJNkMMgeAUJj4dLkMgQswtOpvU4+
MrwXIFctQMsBg3Vsxog7Hqbr3UjsX4DD+/+UWl9kPDzAiK2UViopnK4SiDcqiI2p
pG7G7sgyPVOsTSsdftRFdrBNfmfkjSQoBQqIeW9YtNnAAEULbt+3If5KJTIeA+W/
BYmOldLGp38bx47OPzw2UUYL+GAPw7J3v9CyMBZbzb62hBXCZKdQadqL46U6IgGi
3cVmJ0qFALEXeIoJqmp6HzLL49ASeUGnVH5olfbX47+2H0aZjgM9sQpfpSzQZsbg
IS1HH6Fip/2ZtEtAjQIDAQABo4IC0jCCAs4wDgYDVR0PAQH/BAQDAgWgMB0GA1Ud
JQQWMBQGCCsGAQUFBwMBBggrBgEFBQcDAjAMBgNVHRMBAf8EAjAAMB0GA1UdDgQW
BBQIRDB+G6cygmQfReWAId1hgcrNtTAfBgNVHSMEGDAWgBTAzANGuVggzFxycPPh
LssgpvVoOjB3BggrBgEFBQcBAQRrMGkwMgYIKwYBBQUHMAGGJmh0dHA6Ly9vY3Nw
LnN0Zy1pbnQteDEubGV0c2VuY3J5cHQub3JnMDMGCCsGAQUFBzAChidodHRwOi8v
Y2VydC5zdGctaW50LXgxLmxldHNlbmNyeXB0Lm9yZy8wfwYDVR0RBHgwdoIcKi5h
cHAuYXJrYWRpLmRldi5zdXBlcmh1Yi5pb4IdKi5hcHBzLmFya2FkaS5kZXYuc3Vw
ZXJodWIuaW+CGmFwcC5hcmthZGkuZGV2LnN1cGVyaHViLmlvghthcHBzLmFya2Fk
aS5kZXYuc3VwZXJodWIuaW8wTAYDVR0gBEUwQzAIBgZngQwBAgEwNwYLKwYBBAGC
3xMBAQEwKDAmBggrBgEFBQcCARYaaHR0cDovL2Nwcy5sZXRzZW5jcnlwdC5vcmcw
ggEFBgorBgEEAdZ5AgQCBIH2BIHzAPEAdgCwzIPlpfl9a698CcwoSQSHKsfoixMs
Y1C3xv0m4WxsdwAAAWl3QtA0AAAEAwBHMEUCIHfiyjl7cHLRJl0CqtcJdEjlK9V7
glbtTleI8oDlrhoXAiEAgi/cKm1f2RRHdpdvUWfcmN++GVYiDZ4nHDxZCN3oOlgA
dwDdmTT8peckgMlWaH2BNJkISbJJ97Vp2Me8qz9cwfNuZAAAAWl3QtmqAAAEAwBI
MEYCIQDnd7ztqcXHPO2Hm6kIO04ivh6CvZaCZSZskB47leZhWwIhALttL8qNeDzo
jYRMpHbmmyIUIsvzuCmqwe6zoDxmCIqkMA0GCSqGSIb3DQEBCwUAA4IBAQBCfAjB
rt6vvzWvriBYZ2pqLylFx68w4thsEjIhh2eAOVUWP2RYaLI8mFFAg3zy7tSC2357
d6/nm135PRPuMx5tkf5KjjNABl2yQeOrJicBeM8+knjWILxu03uowQTIvDMWGmE6
Fm0zKTl/fOJJFYADZGk9pU4vERUlG7qgVS/SHrFnfnqxS3x18w/Km9qYV1bQZW7l
OpjjR/myscqUJ4sM+5Ah2ymfFniIeYaK/zQQTJE4jmadwlYrZMHni6GNZtYpDhBd
mZRPEKlACZP9vwEgjYoD9vfKko/VcQTvbvk071A3M7U5DKzZdgUm6PU745w9jKfZ
UBVL/UwarPvchxOm
-----END CERTIFICATE-----
```

**Comment by traefiker** [comment]
Hi! I'm Træfiker :robot: the bot in charge of communication regulation.

Thanks for your interest in Traefik!

Issue templates help us help you by providing all necessary information.

Please edit your issue and use the available templates:

- [Feature request](https://raw.githubusercontent.com/containous/traefik/master/.github/ISSUE_TEMPLATE/Feature_request.md)
- [Bug report](https://raw.githubusercontent.com/containous/traefik/master/.github/ISSUE_TEMPLATE/Bug_report.md)

And remember: each time someone ignores the template, a cute little bunny dies.
<details>
<summary>#SaveTheCuteBunny :heart: :rabbit: :heart:</summary>

![southpark - estrella kill rabbit](https://user-images.githubusercontent.com/5674651/27143914-3cea7896-5130-11e7-81f8-02d8ac2da7a7.gif)

![southpark - estrella kill rabbit2](https://user-images.githubusercontent.com/5674651/27143924-49104c72-5130-11e7-82ae-ccedbd5dfa1c.gif)

</details>

**Comment by ldez** [comment]
Closed by #4821
